lib/fizzy/asserts.hpp

```diff
@@ -0,0 +1,28 @@
+// Fizzy: A fast WebAssembly interpreter
+// Copyright 2020 The Fizzy Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <cassert>
+
+#ifndef __has_builtin
+#define __has_builtin(name) 0
+#endif
+
+#ifndef __has_feature
+#define __has_feature(name) 0
```

Comment:
Not used?

Reply:
Used.

Reply:
Only in tests.

Reply:
But in 2 files.

Reply:
This is standard implementation of `__has_feature()`. It could be placed somewhere in tests I don't want to create new file for it. Here is good enough I think.